### PR TITLE
DOP-4842: Sidenav init Dark Mode

### PR DESCRIPTION
**Pull request by mmeigs**
### Stories/Links:

DOP-4842

### Current Behavior:

[Atlas](https://www.mongodb.com/docs/atlas/getting-started/)
[Manual](https://www.mongodb.com/docs/manual/)

### Staging Links:

[Atlas](https://docs-mongodb-org-stg.s3.us-east-2.amazonaws.com/master/cloud-docs/matt.meigs/DOP-4842-init-sidenav/getting-started/index.html)
[Manual](https://docs-mongodb-org-stg.s3.us-east-2.amazonaws.com/v7.0/docs/matt.meigs/DOP-4842-init-sidenav/index.html)

### Notes:

This overwrites styles with css variables for the Sidenav to init in Dark Mode without a flash of Light Mode.

### README updates

- - [ ] This PR introduces changes that should be reflected in the README, and I have made those updates.
- - [x] This PR does not introduce changes that should be reflected in the README

### CODE REVIEW

File: src/components/Sidenav/Sidenav.js [hunk 1]
```diff
@@ -110,6 +110,29 @@ const SidenavContainer = styled.div(
...
+      color: var(--sidenav-item-color);
+    }
+
+    [data-testid='side-nav-collapse-toggle'] {
+      background-color: var(--sidenav-toggle-bg-color);
+      color: var(--sidenav-toggle-color);
+      border-color: var(--sidenav-toggle-border-color);
+    }
+
+    a[aria-current='page'] {
```
Comment (seungpark) [inline src/components/Sidenav/Sidenav.js:128]: sweet! can we also do this for IA component? seems like docs landing is being overwritten to black:

image

**Review by seungpark — APPROVED** [review]
this looks great! thought it would be much more complicated but very clean! minor notes on minimizing repeated light/dark values and one note on IA component below!

**Review by seungpark — CHANGES_REQUESTED** [review]
! sorry meant to require changes beforehand (re IA item)

File: src/components/Sidenav/Sidenav.js (hunk 1 above)
Comment (mmeigs) [reply src/components/Sidenav/Sidenav.js:128]: I've been leaving IA to be dealt with in the landing pages ticket.
I left a comment on that ticket linking to this: https://jira.mongodb.org/browse/DOP-4847?focusedCommentId=6622233&focusedId=6622233&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-6622233

**Review by seungpark — APPROVED** [review]
LGTM !

**Pull request merged**